cmake/Findcppzmq.cmake

```diff
@@ -0,0 +1,56 @@
+#[=======================================================================[.rst:
+Findcppzmq
+-----------
+
+Finds cppzmq. Does not (currently) support version matching.
+
+IMPORTED Targets
+^^^^^^^^^^^^^^^^
+
+The following :prop_tgt:`IMPORTED` targets may be defined:
+
+``cppzmq::cppzmq``
+  cppzmq.
+
+Result variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables in your project:
+
+``cppzmq_FOUND``
+  true if cppzmq headers were found
+``cppzmq_INCLUDE_DIR``
+  the directory containing cppzmq headers
+
+TODO
+^^^^
+
+* Version matching
```

Comment:
Same here.

Reply:
I'll probably do this when I move the find module into the irods repository.